src/layout/Flex.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+
+// TODO: align items/content/justify/etc support
+export let Flex = ({ style, ...props }) => (
+  <div style={{ display: 'flex', ...style }} {...props} />
+)
```

Comment:
We can pretty much make this better with out marketplace's Flex thing.

Reply:
The one in marketplace is dependent on bootstrap and is why I didn't use it. I didn't need the other props yet so that's why it's marked as a TODO.